core/schema/src/main/resources/org/hawkular/schema/bootstrap.groovy

```diff
@@ -55,19 +55,20 @@ if (keyspace.equals("hawkulartest") || keyspace.equals("hawkular_metrics_rest_te
   keyspaceCreate = "$keyspaceCreate AND DURABLE_WRITES = 'false'"
 }
 
-executeCQL(keyspaceCreate)
-executeCQL("USE $keyspace")
+if(createSchema) {
```

Comment:
There is another reason we do not want ```createSchema```. In the event that ```resetDB``` is true and ```createSchema``` is false, then schema installation breaks. Each of the commands in this script is idempontent. It is safe for them to be run multiple times.